pyOCD/coresight/ap.py

```diff
@@ -337,8 +337,10 @@ def _handle_error(self, error, num):
 
 class AHB_AP(MEM_AP):
     def init(self, bus_accessible=True):
-        super(AHB_AP, self).init(bus_accessible)
+        # Init super first with bus access disabled, so it doesn't read the ROM table.
+        super(AHB_AP, self).init(False)
```

Comment:
No need to do it now. Thanks for the quick response in optimizing the rom table read!